situp-plugins/service-map-stateful/src/main/java/com/amazon/situp/plugins/processor/ServiceMapProcessorConfig.java

```diff
@@ -3,6 +3,5 @@
 public class ServiceMapProcessorConfig {
     static final String WINDOW_DURATION = "window_duration";
     static final int DEFAULT_WINDOW_DURATION = 180;
-    static final String DEFAULT_TRACES_LMDB_PATH = "data/service-map/spans";
-    static final String DEFAULT_SPANS_LMDB_PATH = "data/service-map/traces";
+    static final String DEFAULT_LMDB_PATH = "data/service-map/";
```

Comment:
```
~/open-source/simple-ingest-transformation-utility-pipeline/data master*
❯ ls 
service-map/                                 service-mapdb-1602165692694-prev             service-maptrace-db-1602165692695-lock
service-mapdb-1602165692085                  service-mapdb-1602165692694-prev-lock        service-maptrace-db-1602165692696-prev
service-mapdb-1602165692085-lock             service-maptrace-db-1602165692695            service-maptrace-db-1602165692696-prev-lock

```

Not sure if this the expected path you want to have?

Reply:
I see, looks like the "/" isnt added onto the directory path. Will fix this.